changelog.txt

```diff
@@ -1,123 +1,314 @@
+	- Add shape definitions to GetarFileReader.
+
+[0.4.1] -- 2017-08-23
+---------------------
+
+Fixed
++++++
 	- Fix minor bug related to QR check for 2d boxes.
 
-0.4.0
+[0.4.0] -- 2017-06-26
+---------------------
 
-	- Added readers/writers:
-		- cif reader
-		- gsd writer
-	- Improved algorithm for the normalization of frames with non-standard box
-	- New supported shape definitions:
+Added
++++++
+	- Add readers/writers:
+		- CIF reader
+		- GSD writer
+	- Support shape definitions:
 		- spheropoly
 		- polyunion
 		- convexpolyhedron_union
-	- The GetarFileReader also reads shape definitions.
-	- Add `gf2pos` script - convert to pos-file from any supported format
-	- Other minor updates:
-		- GetarFileReader skips records that have a non-empty group field.
-		- Interpret the pos-file rotation key word.
-		- Various documentation updates
+	- Add `gf2pos` script - convert to pos-file from any supported format.
+	- Add shape definitions to GetarFileReader.
+	- Interpret the pos-file rotation key word.
+
+Changed
++++++++
+	- Improve algorithm for the normalization of frames with non-standard
+	box.
+	- Various documentation updates.
+	- GetarFileReader skips records that have a non-empty group field.
+
+Version 0.3
+===========
+
+Highlights
+----------
+	- Various documentation improvements.
+	- Add an optimized cythonized DCDFileReader.
+	- Allow trajectory data acess via coherent numpy arrays.
+	- Make snapshot creation and copying HOOMD-blue 2.0 compatible.
+	- The GSDReader now reads velocities.
+	- Support PolyV shape definitions.
 
-0.3.9
+[0.3.9] -- 2017-01-30
+---------------------
 
-	- Fix bug related to trajectory arrays when slicing the array.
+Added
++++++
 	- The GSDReader now reads velocities.
 	- Support PolyV shape definitions.
+
+Changed
++++++++
 	- Update documentation concerning the conversion of rotations from
 	  quaternions to euler angles.
+Fix
++++
+	- Fix bug related to trajectory arrays when slicing the array.
 
-0.3.8:
+[0.3.8] -- 2016-12-21
+---------------------
 
+Fixed
++++++
 	- Hot fix: Negative euler angles were not read correctly in skewed
 	  boxes using the DCDFileReader.
 
-0.3.7:
+[0.3.7] -- 2016-11-07
+---------------------
+
+Added
++++++
+	- Add the 'whence' argument to the file format's seek method.
 
+Fixed
++++++
 	- Fix bug in dcdfilereader leading to incorrect box dimensions
 	  to be read for skewed boxes. Cubic or squared boxes are not
 		affected.
-	- Add the 'whence' argument to the file format's seek method.
 
-0.3.6:
+[0.3.6] -- 2016-10-20
+---------------------
 
+Fixed
++++++
 	- Fix quaternion to euler angle conversion example in the
 		dcd file reader documentation.
 
-0.3.5:
+[0.3.5] -- 2016-09-20
+---------------------
 
+Changed
++++++++
 	- GSDHOOMDFileReader uses the native gsd library if installed.
+	- Reduced warning verbosity.
+
+Fixed
++++++
 	- Fix bug that caused the GSDHOOMDFileReader to ignore dimensions
 		specified in the GSD file.
```

Comment:
Yeah, let's wrap all of these to a single long line.

Reply:
In commit 0391c58